bindings/python/test/test_pandas.py

```diff
@@ -98,13 +99,21 @@ def test_aggregate_simple(self):
         self.assertEqual(agg_cmd.command["pipeline"][0]["$project"], projection)
         self.assertEqual(agg_cmd.command["pipeline"][1]["$project"], {"_id": True, "data": True})
 
+    def _assert_frames_equal(self, incoming, outgoing):
+        for name in incoming.columns:
+            col = incoming[name]
+            val = outgoing[name]
+            if str(val.dtype) in ["object", "float64"]:
```

Comment:
A comment here or a method docstring would be helpful.

Reply:
Done